src/sentry/receivers/features.py

```diff
@@ -428,6 +429,15 @@ def record_integration_added(integration, organization, user, **kwargs):
         provider=integration.provider,
         id=integration.id,
     )
+    metrics.incr(
+        "integration.added",
+        sample_rate=1.0,
+        tags={
+            "integration": integration.provider,
+            "integation_id": integration.id,
```

Comment:
I wasn't sure if we wanted to log anything besides just the provider since we're only concerned with number of installs per integration here.

Reply:
@ceorourke Metrics are not the same thing as logging. Adding different tags has a cost associated with it. We should only have tags that don't have too many values. So we should just not have a tag for the `integration_id` and `organization` since we wouldn't use it in Datadog. If we want logging here, we should do that separately.